CHANGELOG.md

```diff
@@ -3,8 +3,81 @@
 <!-- ## Unreleased -->
 ### Added
 
+#### Plugin updates and additions ([#233](https://github.com/Shopify/eslint-plugin-shopify/pull/233))
+
+#### Breaking Changes
+
+* `shopify/jquery-dollar-sign-reference` has been removed.
+
+* The `ava`, `mocha`, `jquery`, `lodash`, `typescript-prettier`, `typescript-react` rulesets have been removed.
```

Comment:
Would it be worth showing what you should use instead of `typescript-prettier` and `typescript-react`?

Something like:

```md
* The `ava`, `mocha`, `jquery` and `lodash` rulesets have been removed as these tools are are not commonly used at Shopify.
* The `typescript-react` and `typescript-prettier` rulesets have been removed. Replace `["plugin:shopify/typescript-react"]` with `["plugin:shopify/typescript", "plugin:shopify/react"]` and replace`["plugin:shopify/typescript-prettier"]`  with `["plugin:shopify/prettier"]`
```